scripts/deploy-helm.sh

```diff
@@ -286,3 +286,14 @@ fi
 if [[ $DEPLOYMENT_TYPE == metrics ]] || [[ $DEPLOYMENT_TYPE == all ]]; then
   echo "kubectl -n dynatrace logs -l app=dynatrace-gcp-function -c dynatrace-gcp-function-metrics"  > ${CMD_OUT_PIPE}
 fi
+
+if [[ $DEPLOYMENT_TYPE == all ]]; then
```

Comment:
please add comment explaining why we cannot construct LOG_VIEWER url for deployment type = 'logs'